ci/integration-tests-windows.groovy

```diff
@@ -27,6 +27,8 @@ node('mesos-ubuntu') {
             checkout scm
             sh 'make windows'
             stash includes: 'build/windows/**', name: 'dcos-windows'
+            sh 'wget https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe'
```

Comment:
It'd be nice to have a comment indicating why we're doing this. For anyone who is not aware of the current HTTPS issues in our windows node it wouldn't be clear why we are stashing from here.

Reply:
OK, adding that as a comment in 1.12/1.13/14 branches in a commit on top later today.